make/prologue/paths.mk

```diff
@@ -0,0 +1,29 @@
+# add-path: Append custom paths for non-system software.
+define add-path
+ifneq ("$(wildcard $(1)/lib64)","")
+  INCLUDE_FLAGS += -I$(1)/include -L$(1)/lib64
+  CHPL_FLAGS    += -I$(1)/include -L$(1)/lib64 --ldflags="-Wl,-rpath,$(1)/lib64"
+endif
+INCLUDE_FLAGS += -I$(1)/include -L$(1)/lib
+CHPL_FLAGS    += -I$(1)/include -L$(1)/lib --ldflags="-Wl,-rpath,$(1)/lib"
+endef
+# Usage: $(eval $(call add-path,/home/user/anaconda3/envs/arkouda))
```

Comment:
you are probably aware, but this is going to need to change after https://github.com/Bears-R-Us/arkouda/pull/5145